packages/lending/src/adapter.config.ts

```diff
@@ -0,0 +1,61 @@
+import { arbitrumTokens, avalancheTokens, mainnetTokens, metisTokens, optimismTokens, polygonTokens } from './tokens';
+import * as common from '@protocolink/common';
+
+interface Config {
+  chainId: number;
+  primaryStablecoin: common.Token;
+  secondaryStablecoin: common.Token;
+  primaryNonstablecoin: common.Token;
+}
+
+export const configs: Config[] = [
+  {
+    chainId: common.ChainId.mainnet,
+    primaryStablecoin: mainnetTokens.USDC,
+    secondaryStablecoin: mainnetTokens.DAI,
+    primaryNonstablecoin: mainnetTokens.ETH,
+  },
+  {
+    chainId: common.ChainId.polygon,
+    primaryStablecoin: polygonTokens.USDC,
+    secondaryStablecoin: polygonTokens.DAI,
+    primaryNonstablecoin: polygonTokens.WETH,
+  },
+  {
+    chainId: common.ChainId.arbitrum,
+    primaryStablecoin: arbitrumTokens['USDC.e'],
+    secondaryStablecoin: arbitrumTokens.DAI,
+    primaryNonstablecoin: arbitrumTokens.ETH,
+  },
+  {
+    chainId: common.ChainId.optimism,
+    primaryStablecoin: optimismTokens.USDC,
+    secondaryStablecoin: optimismTokens.DAI,
+    primaryNonstablecoin: optimismTokens.ETH,
```

Comment:
Could I ask what are the meanings of `primaryStablecoin`, `secondaryStablecoin` and `primaryNonstablecoin`?

Reply:
We currently use stablecoins as collateral tokens for leverage in both long and short positions.

When the leverage token is not USDC, we use `primaryStablecoin` as the collateral token. In all other cases, `secondaryStablecoin` is used as the collateral token.

`primaryNonstablecoin` as a non-stablecoin, common token.